view/base/web/template/content/json.html

```diff
@@ -0,0 +1,12 @@
+<pre data-bind="text: value"
+style="background: #f6f6f6;
+    border: 1px solid #cccccc;
+    color: #111111;
+    line-height: 1.42857143;
```

Comment:
This seems super-specific...

Reply:
10/7, for some reason?

Reply:
So some quick googling tells me this comes about from wanting a line height of 20px when the font-size is 14px (20/14 = 1.42857143). I feel like if you want a specific line height, it should just be set to that. (especially since the font-size is set to 1.2rem and not 14px)

I'd be interested in hearing from a front-end on this one

Reply:
cc @luke-denton-aligent

Reply:
yeah i've realised i've copied over some generated css. i'll update this

Reply:
i've simply set it to `1.5`. let me know if it should be otherwise.

Reply:
@aligent-lturner The reason for setting a value like this would be to retain the aspect ratio. If a user increases their font size in the browser, then having this unit less value means the line height will also respond accordingly. I agree that 8 decimal places is probably a little much, and I do wonder if we could just set the `line-height` to be `calc(20/14)`.. That way the aspect ratio is super clear

Reply:
right, thanks. i've updated the line height as per @luke-denton-aligent's suggestion. `calc(20/14)`